README.md

```diff
@@ -77,16 +77,16 @@ What's collected
 ----------------
 
 The following list documents the types of metrics collected by this agent. Be aware, this is
-only a summary, not every specific data point.  It is possible that some PII type information 
```

Comment:
So this one is big more tricky.     It's like "UPC code":  Universal Product Code code....

I threw "type" in there to *avoid* the "Personally Identifiable information information" redundancy.  But I'm not sure that PII is known widely enough to stand alone.  Also, much of this data isn't *technically* PII, it could be PII or PII-ish?....

I don't know.  Feel free to tell me that I'm overthinking this.

If you can think of a better option given any of this feedback go for it, otherwise it's okay as is.